src/libcore/fmt/mod.rs

```diff
@@ -844,12 +844,36 @@ impl Display for char {
     }
 }
 
+// Two extra bytes for 0x
+#[cfg(target_pointer_width = "32")]
+const POINTER_PADDING: Option<usize> = Some(10);
+#[cfg(target_pointer_width = "64")]
+const POINTER_PADDING: Option<usize> = Some(18);
```

Comment:
Wherever possible I tend to prefer `if cfg!(...)` over `#[cfg]`, and it looks like for all uses of `#[cfg]` here the `cfg!`-based form would suffice?